index.html

```diff
@@ -2995,6 +3024,13 @@ <h2 id="OfflineAudioContext">
             </dt>
             <dd>
               <p>
+                If the <a>relevant settings object</a>'s <a>responsible +
```

Comment:
Same comment as above for "relevant settings object" and "responsible document".